index.html

```diff
@@ -17,21 +17,30 @@
                     <section>
                             <header class="navbar">
                                 <a href="#">
-                                    <h4>Sandeep</h4>
+                                    <h4>My Logo</h4>
                                 </a>
                                 <nav>
+                                    
+                                    <ul id="desktop_menu_list">
+                                        <li><a class="menu_link" href="#work">Portfolio</a></li>
+                                        <li><a class="menu_link" href="#about">About</a></li>
+                                        <li><a class="menu_link" href="#contact_section">Contact</a></li>
+                                        <li><a class="menu_link" href="#email" ><img src="images/mail_img.png" alt="mail image"></a></li>
+                                    </ul>
+                                    
                                     <div id="bar">
                                         <img src="images/Menu.png" alt="icon">
                                         <img src="images/Menu.png" alt="icon">
                                         <img src="images/Menu.png" alt="icon">
                                     </div>
+                                
                                 </nav>
                             </header>
                     </section>
                         <!-- headline section -->
-                    <section>
+                    <section class="head_class">
                             <div class="headline">
```

Comment:
- [ ] Kindly reduce the margin beneath the button, and center the white line at the bottom.

- Your design
![image](https://user-images.githubusercontent.com/41163571/230712485-8459ec47-2322-4bdd-b61d-33d39f40874f.png)

Reply:
Done.